core/providers/google_music.py

```diff
@@ -25,3 +25,5 @@ def get_music_name(self, url):
         artist = soup.find('div', class_='album-artist fade-out')
         if title and artist:
             return f'{artist.text} - {title.text}'
+
+        return soup.find('meta', property='og:title').get('content')
```

Comment:
What if Soup will not find anything here?

Reply:
And check please, maybe that meta can be found in other links to GP Music as well, and we can use only that approach

Reply:
@Ignisor +